src/index.html

```diff
@@ -7,12 +7,63 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+      <p class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__prod-code">Product code: 195434</p>
+      <div class="card__block">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card-review">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__txt">Price:</p>
+        <p class="card__money">$2,199</p>
+      </div>
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price">
        <p class="card__txt">Price:</p>
        <p class="card__money">$2,199</p>
        
      </div>
```

Reply:
Still not fixed